torch/_prims/rng_prims.py

```diff
@@ -0,0 +1,51 @@
+from typing import Tuple
+
+import torch
+from torch import _prims
+from torch._decomp.decompositions_for_rng import RNGStateHelper, throw_on_cpu
+from torch.types import _device, _dtype
+
+
+def _philox_rand(
```

Comment:
The contract for prims is pretty strict. Is that really a new prim we want to add?
cc @mruberry

Reply:
I took inspiration from TorchInductor, but happy to add a new namespace if thats better -

Reply:
At the very worst case, these should just end up being analogous to nvprims.